app/main.py

```diff
@@ -2,24 +2,61 @@
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, instance, name: str) -> None: # noqa
```

Comment:
noqa? why you need it? delete please

Reply:
Do you can explain me how write correct type hint for descriptor's methods?

Reply:
`object ` is a good idea for annotation